src/prefect/client/orchestration/__init__.py

```diff
@@ -901,7 +901,7 @@ async def delete_task_run(self, task_run_id: UUID) -> None:
         Delete a task run by id.
 
         Args:
-            task_run_id: the task run ID of interest
+            task_run_id: The unique identifier of the task run to retrieve.
```

Comment:
this is the `delete_flow_run` method

```suggestion
            task_run_id: The unique identifier of the task run to delete.
```